### PR TITLE
FEAT : Date, filter notes

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
- Add the Date to the note , When its created.
- Filter notes with labels.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -17,9 +17,11 @@
         "jwt-encode": "^1.0.1",
         "miragejs": "^0.1.42",
         "mockman-js": "^1.0.5",
+        "moment": "^2.29.2",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "react-masonry-css": "^1.0.16",
+        "react-moment": "^1.1.2",
```
Comment (vijaytembugade) [inline package-lock.json]: You are using both , `react-moment` and `moment`, any perticular reason?

File: src/frontend/pages/LabelsPage/LabelsPage.js [hunk 2]
```diff
@@ -14,10 +14,19 @@ const LabelsPage = () => {
     return [...acc, ...curr.tags];
   }, []);
   const uniqueTags = [...new Set(newLabels)];
-  const [uniqueState, dispatch] = useReducer(reduceTags, []);
   const reduceTags = (state, action) => {
     console.log(state, action);
+    switch (action.type) {
+      case "FILTER_BY_TAG":
+        console.log("from reducer : ", action.payload);
+      // return state.filter((note) => note.tags.includes(action.payload));
```
Comment (gopalbbharadva) [inline src/frontend/pages/LabelsPage/LabelsPage.js]: Remove consoles.

File: src/frontend/pages/LabelsPage/LabelsPage.js [hunk 3]
```diff
@@ -38,7 +47,13 @@ const LabelsPage = () => {
                   {uniqueTags.map((tag, i) => (
                     <li key={i} className="tag-btn">
                       <label htmlFor={tag}>
```
Comment (gopalbbharadva) [inline src/frontend/pages/LabelsPage/LabelsPage.js]: Suggestion: Instead of i, you can name specifically index so it will be more readable.

**Review by omgaikwad — APPROVED** [review]
Clean code, good file structure, you are good to go!

**Review by vijaytembugade — APPROVED** [review]
Looks good. Functionality and also workingf ine. Good work.

**Review by prathmeshjagtap — APPROVED** [review]
Code is Good and Clean , Learned how to use moment to sort by time, nice implementation

**Review by gopalbbharadva — APPROVED** [review]
Nice UI, Hamburger  drawer is awesome.

File: package-lock.json (hunk 1 above)
Comment (MiheerTamkhane) [reply package-lock.json]: No I forgot to remove react-moment

File: src/frontend/pages/LabelsPage/LabelsPage.js (hunk 3 above)
Comment (MiheerTamkhane) [reply src/frontend/pages/LabelsPage/LabelsPage.js]: yupp Thank you I'll Do

File: src/frontend/pages/LabelsPage/LabelsPage.js (hunk 2 above)
Comment (MiheerTamkhane) [reply src/frontend/pages/LabelsPage/LabelsPage.js]: yup thanks

**Pull request merged**